.github/workflows/ci.yaml

```diff
@@ -298,4 +299,9 @@ jobs:
                   --skip test_interpreter_crashes_issue1271_set_in_if \
                   --skip test_interpreter_models_assign_equal_model \
                   --skip example_carousel \
-                  --skip example_fancy_demo
+                  --skip example_fancy_demo \
+                  --skip test_interpreter_elements_path_fit \
+                  --skip test_interpreter_layout_path \
+                  --skip test_interpreter_layout_path_alignment \
+                  --skip test_interpreter_layout_path_for \
+                  --skip test_interpreter_layout_path_offset
```

Comment:
I believe this works with substring, so --skip test_interpreter_layout_path_ should skip all the path layout ones

Reply:
Ahh, ok, let's try that